### PR TITLE
updates clusterNodes routeRoles

### PR DESCRIPTION
**Pull request by ChaitanyaVootla**
https://github.com/Shippable/api/issues/15910

### CODE REVIEW

File: migrations/v4.12.1.sql [hunk 1]
```diff
@@ -6189,7 +6189,19 @@ do $$
     perform set_route_role(
       routePattern := '/clusterNodes',
       httpVerb := 'GET',
-      roleCode := 6060
```
Comment (ric03uec) [inline migrations/v4.12.1.sql]: @spsiddarthan why is this removed? does this route not need `justUser` permissions?

Comment (spsiddarthan) [inline migrations/v4.12.1.sql]: @ric03uec, yup - 6060 (justUser) is the right role. Closing this PR.
